mne/viz/topomap.py

```diff
@@ -2488,6 +2445,21 @@ def _plot_corrmap(data, subjs, indices, ch_type, ica, label, show, outlines,
                      contours=contours, show=False, image_interp='bilinear')[0]
         _hide_frame(ax)
     tight_layout(fig=fig)
+    
+    default_fig_adjust = dict(left=None, bottom=None, right=None, top=0.8, wspace=None, hspace=None)
```

Comment:
You can use the builtin dictionary method `setdefault` here as in `fig_adjust.setdefault('left', None)` which will add left to the dictionary with `None` as the value if it's not in the dictionary already.